docker-compose-dev.yaml

```
@@ -0,0 +1,33 @@
+services:
+  app:
+    container_name: single_cell
+    image: gcr.io/broad-singlecellportal-staging/single-cell-portal:development
+    environment:
+      VITE_RUBY_HOST: vite
+    depends_on:
+      - vite
+    env_file:
+      - config/secrets/.source_env.bash
+    entrypoint: ./rails-dev-entrypoint.sh
+    ports:
+      - 3000:3000
+      - 3001:3001
+    expose:
+      - 27017
+    volumes:
+      - .:/home/app/webapp
+      - type: volume
+        target: /home/app/webapp/node_modules
+  vite:
+    container_name: single_cell_vite
+    image: gcr.io/broad-singlecellportal-staging/single-cell-portal:development
+    entrypoint: ./vite-dev-entrypoint.sh
```

For internal observability:

```suggestion
    entrypoint: ./vite-dev-entrypoint.sh --vite-env-var VITE_DEV_SETUP="docker-compose"
```

---

[0ed7c91](https://github.com/broadinstitute/single_cell_portal_core/pull/1641/commits/0ed7c91d3d2fbbf8d18aed7fdfbc1ab1b76322d7)